pkg/kubelet/cm/memorymanager/memory_manager_test.go

```diff
@@ -1896,6 +1897,13 @@ func TestRemoveContainer(t *testing.T) {
 	}
 }
 
+func getPolicyNameForOs() policyType {
```

Comment:
Could you add a comment explaining what's supported on windows?
Also, I'm not familiar with the tests, but are all the tests still meaningful if we use the best-effort policy?